src/main/java/domain/Line.java

```diff
@@ -1,51 +1,26 @@
 package domain;
 
-import java.util.ArrayList;
 import java.util.List;
-import java.util.Random;
+import java.util.stream.Collectors;
 
 public class Line {
-    public static final String LESS_THEN_MIN_PERSON_NUMBER_MESSAGE = "최소 2명의 참가자가 필요합니다.";
-    public static final int MIN_PERSON_NUMBER = 2;
-    public static final int ZERO = 0;
+    private final Points points;
 
-    private final List<Boolean> points = new ArrayList<>();
-    private final Random random = new Random();
-
-    public Line(int countOfPerson) {
-        validatePersonNumber(countOfPerson);
-        generatePoints(countOfPerson);
-    }
-
-    private void generatePoints(int countOfPerson) {
-        for (int i = 0; i < countOfPerson - 1; i++) {
-            addPoint(i);
-        }
-    }
-
-    private void addPoint(int currentIndex) {
-        if (isFirstIndex(currentIndex) || isBeforeValueThatFalse(currentIndex)) {
-            points.add(random.nextBoolean());
-            return;
-        }
-        points.add(false);
-    }
-
-    private boolean isBeforeValueThatFalse(int currentIndex) {
-        return !points.get(currentIndex - 1);
+    protected Line(List<Boolean> points) {
+        this.points = new Points(points.stream()
+                .map(Point::new)
+                .collect(Collectors.toList()));
```

Comment:
코멘트를 드렸던 부분이 오히려 헷갈릴 수도 있다고 생각이드네요 😭 
다름이 아니라 전 `Line` 객체가 Points를 상태 값으로 들고 있으면 생정자에서 받을 때 `Points` 를 받으면 되지 않을까 라고 생각을 했습니다.
생성자는 객체가 생성될때 초기화 해주는데 `Line ` 생성자에서 `Points` 객체를 생성하는게 맞을까? 라는 생각이 들었습니다.

```suggestion
    protected Line(Points points) {
        this.points = points;
```

가 될 수 있고 해당 책임을 `DefaultlineGenerator` 에서 Points 객체를 만들 수 있지 않을까요?